smf/switch_zone_setup/manifest.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+
+<service_bundle type='manifest' name='switch_zone_setup'>
+
+<service name='system/illumos/switch_zone_setup' type='service' version='1'>
+  <create_default_instance enabled='false' />
```

Comment:
Why is this not enabled by default?  I would expect `true` here, so that the manifest just starts when the zone boots up.

Reply:
So the current model is that all the services start disabled and that sled-agent's bootstrap process imports them, provides inputs (e.g. specifies ports), and then enables them. For this script specifically there are no inputs that sled-agent needs to provide, so we could go either way, but I was thinking about uniformity with the other services. What do you think?

Reply:
Note that even if we set `enabled='true'` the manifest isn't automatically imported, so the script won't run when the zone boots up. Instead, sled-agent imports the manifest explicitly.

Reply:
By placing it in `/var/svc/manifest` it actually will be imported automatically by the `manifest-import` service.

Because it's a short term (touch wood) workaround for a missing feature in the brand I'd be inclined not to make sled agent aware of it if possible.  It's also in line with where we _want_ to be, which is that sled agent doesn't reach into the zone to work around with SMF; i.e., that zone images are as self assembling as possible.

Reply:
OK, so we discussed this in #oxide-q&a on Matrix, and here's what we came to understand:

1. I ran into what seemed like an issue once where the `switch_zone_setup` script appeared not to start. I thought that it meant that manifests had to be imported explicitly.
2. My incorrect belief was reinforced by me finding the bootstrap code which imports all the site-specific manifests. In reality, the bootstrap code doesn't 

I removed all the sled-agent code I'd added in the earlier version and tried again. In 5 separate tries I was unable to reproduce my issue from earlier: each time, the setup script automatically ran in the switch zone and the wicket user was created.

I'm going to chalk this up to me screwing up the manifest.xml somehow on the one try that I did where the manifest didn't get imported, though I guess that there's a small possibility there's a nondeterministic bug somewhere in the system.